### PR TITLE
Issue #1705 Add -f to delete commands

### LINKED ISSUE #1705: Add '-f' shortcut the various delete commands
Atm the delete commands require an explicit long flag version of '--force', eg

    $ minishift delete --force
    $ minishift profile delete --force

These _force_ flags should also be available under the short '-f' form.

### PR DESCRIPTION
**Pull request by bhavin192**
Fixes #1705

Added `-f` shorthand to the `delete` and `profile delete` commands.
Please let me know if I have missed any command.

### CODE REVIEW

**Comment by gbraad** [comment]
add to whitelist

File: cmd/minishift/cmd/delete.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func removeInstanceConfigs() {
 }
 
 func init() {
-	deleteCmd.Flags().BoolVar(&forceMachineDeletion, "force", false, "Forces the deletion of the VM specific files in MINISHIFT_HOME.")
+	deleteCmd.Flags().BoolVarP(&forceMachineDeletion, "force", "f", false, "Forces the deletion of the VM specific files in MINISHIFT_HOME.")
```
Comment (praveenkumar) [inline cmd/minishift/cmd/delete.go]: 👍

File: cmd/minishift/cmd/profile/profile_delete.go [hunk 2]
```diff
@@ -107,6 +107,6 @@ func runProfileDelete(cmd *cobra.Command, args []string) {
 }
 
 func init() {
-	profileDeleteCmd.Flags().BoolVar(&forceProfileDeletion, "force", false, "Forces the deletion of profile and related files in MINISHIFT_HOME.")
+	profileDeleteCmd.Flags().BoolVarP(&forceProfileDeletion, "force", "f", false, "Forces the deletion of profile and related files in MINISHIFT_HOME.")
```
Comment (praveenkumar) [inline cmd/minishift/cmd/profile/profile_delete.go]: 👍

**Comment by budhrg** [comment]
Working fine :+1: 
```
$ minishift delete -f
Deleting the Minishift VM...
Minishift VM deleted.
```

**Comment by budhrg** [comment]
Thanks @bhavin192 for the PR.

**Pull request merged**